scripts/variantstore/analysis/sharding/get_workflow_metadata.sh

```diff
@@ -0,0 +1,12 @@
+WF_ID=$1
+TOKEN=$(gcloud auth print-access-token)
+
+PARAMS="includeKey=start&includeKey=end&includeKey=executionStatus&includeKey=preemptible&includeKey=shardIndex&includeKey=wdl-task-name&includeKey=jes&includeKey=jobId&includeKey=executionEvents&includeKey=outputs"
+
+# for BIG workflows where we are going to get PAPI info for start/stop times anyway
+PARAMS="includeKey=start&includeKey=end&includeKey=executionStatus&includeKey=preemptible&includeKey=shardIndex&includeKey=wdl-task-name&includeKey=jes&includeKey=jobId"
+
+curl -sL -X 'GET' \
+  "https://api.firecloud.org/api/workflows/v1/${WF_ID}/metadata?${PARAMS}" \
```

Comment:
not sure if you want to replace this with what I was able to get to work or just document it as an option, but I ran
```
"https://rawls.dsde-prod.broadinstitute.org/api/workspaces/${WORKSPACE_NAME}/submissions/${SUBMISSION_ID}/workflows/${WF_ID}?https://cromwell1-int-lb.dsde-prod.broadinstitute.org/api/workflows/v1/${WF_ID}/metadata?${PARAMS} \
```
where WORKSPACE_NAME is the name of the workspace including the billing project (e.g. "broad-dsde-firecloud-billing/Genomic Variant Store - GVS Quickstart") and SUBMISSION_ID is the submission ID.

Reply:
I thought I understood this… but are you passing an entire URL after the first `?`?

Reply:
Yes...?  To be completely honest, I found this API call by looking in the browser "Network" tab to see what call the "Workflow dashboard" was making; I don't think it is documented anywhere. But, since a part of the Terra UI (granted, marked as "alpha") uses it and it was the only way I was able to get any metadata for large workflows, I thought it would be helpful to write down.  I would definitely understand if you don't want to add it to anything official.